FlorenceBML/BMLacq38.xml

```diff
@@ -392,20 +394,28 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                             </foreign>
                                                 </desc>
                                 </item>
+                                <item xml:id="e15">
+                                    <locus target="#2r, #5v, #8r, #10v, #11v, #23v, #36r, #46r, #84v, #91r, #98r, #110r, #114v, #117r"/>
+                                    <desc type="Correction">Erasures, corrections and rewrites for example apparently by <ref type="hand" corresp="h2"/></desc>
+                                </item>
                             </list>
                         </additions>
                         <bindingDesc>
-                            <binding contemporary="false" xml:id="binding">
-                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote> 
-                                <decoNote xml:id="b2" type="Cover">European wooden binding, with leather covering the spine and half of the outer plates.</decoNote> 
-                                <decoNote xml:id="b3" type="bindingMaterial">
-                                    <material key="wood"/>
-                                    <material key="leather"/>
-                                </decoNote>
+                            <binding contemporary="non-Ethiopian">
+                                <decoNote xml:id="b1" type="Boards"/>          
                             </binding>
                         </bindingDesc>
                     </physDesc>
                     <history>
+                        <origin>
+                            <origDate notBefore="1508" notAfter="1540"/>
+                            <note>The chronological text goes up to <persName ref="PRS6229LebnaDe"/> </note>
+                            <origDate notBefore="1530" notAfter="1531" cert="medium"/>
+                            <note>According to a note on <locus target="143r"/> the manuscript was produced in the 23rd year of <persName ref="PRS6229LebnaDe"/>
+                                and the "183rd year of Mercy" (= cycle 3 of the Era of the martyrs), that agree with each other correspond to the year 1530 to 1531 
+                                which is also in agreement with the palaeography. Since this note was written most likely by another hand on a bifolium, that was 
+                                probably attached later, it is nevertheless not certain, whether it is precisely correct.</note>
```

Comment:
I do not think the protective bifolium was attached later, I think it is part of the original production unit and was blank and then filled , 
change

```
Since this note was written most likely by another hand on a bifolium, that was 
                                probably attached later 
```
to something like
`Since this text was possibly inserted on the blank folia by another hand ....`